CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 #
 # Build AMICI library
 #
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
What was that for if there is no other change in CMake files?

Reply:
@paszkow pointed out that using the option `--parallel` requires at least 3.12., this is why I just put it...

Reply:
Hmm... that would have to be checked in the build script. Here it is too late.

Reply:
okay, I see. ... yes, okay. I'll remove for the moment. Let's do that later and not mix up too many things.